### PR TITLE
Add "round" to standard filters

### PR DESCRIPTION
**Pull request by AdManteza**
- [x] add `{{ foo | round }}` to standard filters

### CODE REVIEW

File: lib/liquid/standardfilters.rb [hunk 1]
```diff
@@ -259,6 +259,13 @@ def modulo(input, operand)
       apply_operation(input, operand, :%)
     end
 
+    def round(input, n = 0)
+      result = to_number(input).round(to_number(n))
+      result = result.to_f if result.is_a?(BigDecimal)
+      result = result.to_i if n == 0
+      result
+    end
```
Comment (AdManteza) [inline lib/liquid/standardfilters.rb:267]: got this from the initial implementation of this filter [here](https://github.com/Shopify/liquid/commit/2239921804b40707f5e694942e8dd664d43a6912), 4 years ago 😛

Comment (jasonewall) [reply lib/liquid/standardfilters.rb:267]: I was wondering if we could add this via a monkey patch in Measure?

Comment (AdManteza) [reply lib/liquid/standardfilters.rb:267]: Hmm maybe inside the `Liquefier::Drop` class [here](https://github.com/yardstick/yardstick/blob/master/lib/liquefier/drop.rb#L1)? Why do we want to monkey patch though? 🤔

Comment (jasonewall) [reply lib/liquid/standardfilters.rb:267]: As a thought experiment to do nasty nasty things with liquid filters later down the road.

Comment (AdManteza) [reply lib/liquid/standardfilters.rb:267]: Copy that. I'll dig into this once MEAS-2842 is settled.

**Comment by AdManteza** [comment]
closing this one. We will add the new filter in Measure's extension of Liquid filters instead.

**Pull request closed without merging**